pkg/utils/utils_test.go

```diff
@@ -0,0 +1,31 @@
+package utils
+
+import "testing"
+
+func TestGetClusterNameFromKubectlCmd(t *testing.T) {
```

Comment:
I see we have pattern here. Can we please write this as table drive test ?

Reply:
Also ordering of string update and call for `GetClusterNameFromKubectlCmd` function is not correct.

Reply:
corrected